components/blitz/resources/omero/api/IAdmin.ice

```diff
@@ -17,9 +17,6 @@ module omero {
 
     module api {
 
-        /**
-         * See <a href="http://hudson.openmicroscopy.org.uk/job/OMERO/javadoc/ome/api/IAdmin.html">IAdmin.html</a>
```

Comment:
These URLs are intended to show the docs that do exist, i.e. http://downloads.openmicroscopy.org/omero/5.1.1/api/ome/api/package-summary.html